CostandacheFlorin/MusicPal
# Feature 1

## Feature description
Create the basic form for sending music recommendations requests

## Expected behaviour
Go to localhost:3000/recommendation and try filling up the form

## To test
Complete the form and check if the error message shows up when you choose more than 5 items.
